packages/strapi-design-system/src/v2/Breadcrumbs/CrumbLink.tsx

```diff
@@ -20,15 +19,10 @@ const StyledLink = styled(BaseLink)`
   }
 `;
 
-export const CrumbLink = ({ children, ...props }) => <StyledLink {...props}>{children}</StyledLink>;
+export interface CrumbLinkProps extends BaseLinkProps {
+  to?: string | undefined;
```

Comment:
```suggestion
```

I'd omit this, this prop isn't on `BaseLink` because it's a react-router-dom prop so this isn't the place for it imo

Reply:
ok